src/main/webapp/data/userMapData.json

```diff
@@ -48,6 +48,7 @@
       "RMIT University",
       "St Catherine's Catholic College",
       "St Ives High School",
+       "St Paul's Anglican Grammar School",
```

Comment:
Fix the indentation here.

Reply:
@whipermr5 Fixed. Thanks for reviewing my PR. 👍